### ISSUE TITLE
Cleanup branches, deployment aftermath

### ISSUE THREAD

**Issue by num46664** [rialtic-community/insight-engine-schema-python#16]
- determine how to get `develop` and `master` to agree while preserving ability to both release to pypi from `master` and also resolve `git+ssh` dependencies from `develop`

**Comment by num46664** [comment]
So here's the state of this:

On a fork of this repo, i merged `master` into `develop` to see if we could still install `git+ssh` dependencies.

The good news: we can without any real changes to the structure

The bad news: by some oversight (on my part), the published package (and subsequently in the pyproject.toml) is named `insight-engine-schema` , where previously it was called `insight-engine-schema-python`

This means that all Pipfiles will fail to resolve the dependency after merging unless we either **1.** change the name of the package in all of them (unfeasable/not worth imo), or **2.** we revert the name of the package, taking down the old one from pypi, and uploading to the new name.

I'm generally in favor of the second option, but lets discuss this on monday and do whatever we have to then.

**Comment by num46664** [comment]
Set this ticket as blocked by another ticket, which encapsulates the work of doing the re-release from `master`. once that is done, This ticket will represent the work involved in merging `master` back into `develop` and troubleshooting any errors that arise.
